### PR TITLE
My journey

### PR DESCRIPTION
**Pull request by Heismanish**
## Description

Created and added my journey in Heismanish.md file
Added my info in Data.json file.

## How Has This Been Tested?

Ran the program and verified it's working.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: journeys/Heismanish.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Manish's Journey
+
+## ☞ The Start!
+
+All this started with from Youtube...
+One day I found randomly came across this guy named Kunal Kuhswaha in a podcast with Ishan Sharma and from there I developed a interest towards not just gsoc a=but about this whole open source thing. Then I started following Kunal Kushwaha and fromm their my coding journey started fron learning Java and getting exposed a very amazing field of DevOps and to open source. 
+
+Though by that time I already had my interest set on Android dev and I got started with my Android journey from Learning kotlin to making some decent projects on Android. And during this time when I was learning Android dev I made a huge mistake of not exploring other field like web dev, DevOps and open source despite not having so much fun with Android dev. It was just a mtter of time before I realised this mistake of mine and soon after that I reconsiderd everything I was doing and decided to explore web-dev and devops and also got started with open source contribution and this realisation was just 1.5 months a ago from today that is 5/10/2022. So yeah now I'm doing web dev and yeah it sure is fun.
```
Comment (vikasganiga05) [inline journeys/Heismanish.md:8]:
```suggestion
Though by that time I already had my interest set on Android dev and I got started with my Android journey from learning kotlin to making some decent projects on Android. And during this time when I was learning Android dev, I made a huge mistake of not exploring other fields like web dev, DevOps, and open source despite not having so much fun with Android dev. It was just a matter of time before I realized this mistake of mine and soon after that I reconsidered everything I was doing and decided to explore web-dev and DevOps and also got started with open source contribution, and this realization was just 1.5 months ago from today, that is 5/10/2022. So yeah, now I'm doing web dev and yeah, it sure is fun.
```

File: journeys/Heismanish.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Manish's Journey
+
+## ☞ The Start!
+
+All this started with from Youtube...
```
Comment (vikasganiga05) [inline journeys/Heismanish.md:5]:
```suggestion
All this started with Youtube...
```

File: journeys/Heismanish.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+One day I found randomly came across this guy named Kunal Kuhswaha in a podcast with Ishan Sharma and from there I developed a interest towards not just gsoc a=but about this whole open source thing. Then I started following Kunal Kushwaha and fromm their my coding journey started fron learning Java and getting exposed a very amazing field of DevOps and to open source.
```
Comment (vikasganiga05) [inline journeys/Heismanish.md:6]:
```suggestion
One day I randomly came across this guy named Kunal Kuhswaha in a podcast with Ishan Sharma and from there I developed an interest in not just gsoc but also in this whole open source thing. Then I started following Kunal Kushwaha and from there my coding journey started from learning Java and getting exposed to a very amazing field of DevOps and open source.
```

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Hi @Heismanish , I suggested some grammar changes. Only change if you like it.

**Comment by Heismanish** [comment]
Thanks @vikasganiga05 for all the help, it was a nice experience contributing this repository.

**Pull request merged**